app/src/androidTest/java/com/br/myfavoritehero/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.br.myfavoritehero
```

Comment:
Is this file necessary?

Reply:
nop, i believe this is generated class...